sale_order_lot_mrp/README.rst

```diff
@@ -0,0 +1,124 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Sale Order Lot MRP
+==================
+
+Allow you to attach a lot in manufacturing order
+that was defined by sale_order_lot_generator module
+for all concerned products ("Lot Generation Auto" = True)
+Name of the MO is also based on sale name.
+
+Usage
+=====
+
+#. First define a bom kit that is composed by components wich are tracked
+by lot and have the option "Lot Generation Auto" = True.
```

Comment:
Add 3 spaces to align with the above start of the sentences so this will show properly.